File: src/main/java/ua/softgroup/matrix/server/supervisor/consumer/json/CurrencyJson.java
package ua.softgroup.matrix.server.supervisor.consumer.json;

import com.fasterxml.jackson.annotation.JsonProperty;

public class CurrencyJson {

    @JsonProperty
    private Integer id;
    @JsonProperty
    private String name;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "CurrencyJson{" +
                "id=" + id +
                ", name='" + name + '\'' +
                '}';
    }
}

File: src/main/scala/ua/softgroup/matrix/server/supervisor/consumer/json/CurrencyJson.scala
package ua.softgroup.matrix.server.supervisor.consumer.json

import com.fasterxml.jackson.annotation.JsonProperty

import scala.beans.BeanProperty

/**
  * @author Oleksandr Tyshkovets <olexandr.tyshkovets@gmail.com>
  */
class CurrencyJson {

  @JsonProperty
  @BeanProperty
  var id: Int = _

  @JsonProperty
  @BeanProperty
  var name: String = _

  override def toString = s"CurrencyJson(id=$id, name=$name)"

}
